profile/README.md

```diff
@@ -0,0 +1,40 @@
+![Unleash_GitHub_Banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
+
+<p align="center">
+<b>The Open Source Feature Flagging Service</b>
+</p>
+
+![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
+
+**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases, and, in the beginning, we were just a few contributors. Together we built a foundation for the amazing community that you see today. 🫂
+
+We love that our community is able to work with our product with all the freedom you could ever want. When you can just test, experiment, and release confidently, you'll end up with a well-rounded product with a ton of awesome interpretations. We're proud of you guys. 🙏
```

Comment:
A few minor suggestions on this paragraph:

I think we should avoid gender-specific terminology where possible. While "guys" is often used in a gender-neutral way, it's also subject to heavy debate. Consider other options where applicable or just drop it completely.

I'm also with Simon on the use of "interpretations" here: it's a little unclear. Could we 

I'm also confused about who "you" are in this sentence. It both appears to refer to the general, unspecified "you" (which even sounds a bit like Unleash) **and** to the community.

I'd suggest rewording this a bit. I've take a stab at it here, but feel free to discard this if you have any better ideas ☺️ 

```suggestion
We love that you — our community — have the freedom to work with our product in any way you want. Together we can test, experiment, and create a well-rounded product that covers a ton of different use cases. We're super proud of you 🙏
```

Reply:
I like this! I spent too long thinking about wording I liked here and came up with nothing, really nice suggestion, sir

Reply:
This is definitely more readable, and puts everything in context. I don't think I can do better than this.